packages/python-name-that-hash/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by amadejpapez <amadej.papez@gmail.com>
+
+pkgname=python-name-that-hash
+_pkgname=Name-That-Hash
+pkgver=1.11.0
+pkgrel=6
+pkgdesc="The Modern Hash Identification System."
+arch=("any")
+url="https://github.com/HashPals/Name-That-Hash"
+license=("GPL3")
+depends=("python" "python-click" "python-rich")
+makedepends=("python-installer" "python-poetry-core" "python-wheel")
+source=("https://files.pythonhosted.org/packages/19/7a/731c96302f39610f1ce6742a4e5b997d621b06f5608dc939f5cd48a639a0/name_that_hash-1.11.0-py3-none-any.whl")
+sha256sums=("59682a35714235d958e723d658c0874abb81edf77ceb9d81d7e6416bc2af3b84")
+
+
+# NOTE: for now install directly via wheel as I was getting the
+# 		ModuleNotFoundError: No module named 'name_that_hash'
+#       error after installation and cannot look into it right now
+
+# build() {
```

Comment:
use https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-PEP517 since there is https://github.com/HashPals/Name-That-Hash/blob/main/pyproject.toml

Reply:
Applied